addon/ng2/tasks/git-init.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var Promise  = require('ember-cli/lib/ext/promise');
+var exec     = Promise.denodeify(require('child_process').exec);
+var path     = require('path');
+var pkg      = require('../package.json');
+var fs       = require('fs');
+var template = require('lodash/string/template');
+
+var gitEnvironmentVariables = {
+  GIT_AUTHOR_NAME: 'angular-cli',
+  GIT_AUTHOR_EMAIL: 'angular-cli@angular339.com',
```

Comment:
@IgorMinar Have any thoughts on which email we should put for the initial commit message of angular-cli projects?

Does Angular have a general contact email?